doc/standard_dems_list.txt

```diff
@@ -0,0 +1,3 @@
+/mnt/pgc/data/elev/dem/tandem-x/90m/mosaic/TanDEM-X_Antarctica_90m/TanDEMX_PolarDEM_90m.tif
+/mnt/pgc/data/elev/dem/gimp/GrIMPv2/data/grimp_v02.0_30m_dem.tif
+/mnt/pgc/data/elev/dem/copernicus-dem-30m/mosaic/global/cop30_tiles_global_wgs84-height_nunatak.vrt
```

Comment:
I'm not a huge fan of putting absolute filepaths in a public repository.  Let's talk about it tomorrow

Reply:
I think this .txt file was originally used as a look up but doesn't actually need to be included. The absolute paths will still be in the geopackage though, so we'll have to resolve the issue there.